Personalizing your Skywallet.md

```diff
@@ -0,0 +1,110 @@
+# Personalizing your Skywallet
+
+After completing the initialization of your Skywallet, you can start to personalize your Skywallet using the Hardware Configuration window in the desktop wallet.
+
+Upon configuring your Skywallet using your desktop wallet, the Skywallet will be listed in the Wallet list.
+
+The list can be viewed when you open your Desktop wallet.
+
+<Screenshot of the list of wallets displayed when the desktop wallet is opened. One of the wallets in the list needs to be a Skywallet>
+
+To personalize your Skywallet, you have to click on the **"Hardware Wallet"** button.
+
+<Screenshot of the desktop wallet highlighting the Hardware wallet button>
+
+Once you click on this button, the following window will appear:
+
+<Screenshot of the Hardware configuration window with the options available when the Skywallet is used after configuration for the first time>
+
+During your first time use of the Skywallet, you will find these two warnings associated with **"Creating a Backup"** and **"Setting a Pin"** being displayed in the Desktop wallet window.
+
+Both of these warnings are to be dealt with seriously or it may compromise the security of your Skywallet.
+
+In order to complete these two tasks, you can use the options in the Hardware Configuration window
+
+Following are the options available in this window:
+
+**Create a backup**
+
+**Create PIN code**
+
+**Wipe the device**
+
+## Create a backup
+
+The Seed of your Skywallet is a key component. Hence, once the initialization is complete it is important to backup this Seed and store it in a secure location.
```

Comment:
Express that this is the way to access all the funds and therfore a key component.

Reply:
Scrap hence. Sounds odd to use it with once

Reply:
There is a seed card that comes with the Skywallet. This needs to be added here. Users should enter the seed on the  seedcard we send.